PubHub/PubHub.BookMobile/ViewModels/LibraryViewModel.cs

```diff
@@ -2,7 +2,8 @@
 using System.Net;
 using CommunityToolkit.Mvvm.ComponentModel;
 using CommunityToolkit.Mvvm.Input;
-using PubHub.BookMobile.ErrorSpecifications;
+using Xamarin.Google.Crypto.Tink.Proto;
```

Comment:
What is this namespace?

Reply:
Don't know, It was there before I did anything to the mobile... I can remove it - seems like it's unused?

Reply:
I removed it.